src/webapi/date.rs

```diff
@@ -1,25 +1,903 @@
+        js!( return @{self.as_ref()}.getUTCHours(); ).try_into().unwrap()
+    }
+
+    /// Returns the milliseconds in the specified date according to
+    /// universal time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCMilliseconds)
+    #[inline]
+    fn get_utc_milliseconds( &self ) -> i32 {
+        js!( return @{self.as_ref()}.getUTCMilliseconds(); ).try_into().unwrap()
+    }
+
+    /// Returns the minutes in the specified date according to universal time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCMinutes)
+    #[inline]
+    fn get_utc_minutes( &self ) -> i32 {
+        js!( return @{self.as_ref()}.getUTCMinutes(); ).try_into().unwrap()
+    }
+
+    /// Returns the month of the specified date according to universal time, as a
+    /// zero-based value (where zero indicates the first month of the year).
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCMonth)
+    #[inline]
+    fn get_utc_month( &self ) -> i32 {
+        js!( return @{self.as_ref()}.getUTCMonth(); ).try_into().unwrap()
+    }
+
+    /// Returns the seconds in the specified date according to universal
+    /// time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getUTCSeconds)
+    #[inline]
+    fn get_utc_seconds( &self ) -> i32 {
+        js!( return @{self.as_ref()}.getUTCSeconds(); ).try_into().unwrap()
+    }
+
+    // DEPRECATED
+    // fn get_year( &self ) -> i32 { }
+
+    /// Sets the day of the Date object relative to the beginning of the
+    /// currently set month.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate)
+    #[inline]
+    fn set_date( &self, day_value: i32 ) {
+        js! { @(no_return)
+            @{self.as_ref()}.setDate(@{day_value});
+        };
+    }
+
+    /// Sets the full year for a specified date according to local time.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setFullYear)
+    #[inline]
+    fn set_full_year( &self, year: i32 ) {
+        js! { @(no_return)
+            @{self.as_ref()}.setFullYear(@{year});
+        };
+    }
+
+    /// Sets the hours for a specified date according to local time, and
+    /// returns the number of milliseconds since 1 January 1970 00:00:00 UTC until the time
+    /// represented by the updated Date instance.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setHours)
+    #[inline]
+    fn set_hours( &self, hour: i32 ) {
+        js! { @(no_return)
+            @{self.as_ref()}.setHours( @{hour} );
+        }
+    }
```

Comment:
The doc comments say that the method returns something, and yet it doesn't. (:

Since the original JavaScript API returns a value here we should *probably* also return a value here. Ditto for other methods like this. (Right now it's not going to be super efficient, but I do have plans to make JS interop cheaper in the future.)